apps/api/src/server/utils/transition-state.js

```diff
@@ -47,7 +47,7 @@ const mapStateMachine = (caseType) => {
 };
 
 /**
- * @param {{caseType: import('@pins/api').CaseType, context: object, status: string | object, machineAction: string, throwError: boolean}} transitionParams
+ * @param {{caseType: import('@pins/api').CaseType, context: object, status: string | undefined, machineAction: string, throwError: boolean}} transitionParams
```

Comment:
Would we ever pass `undefined` for the `status` argument of this function?

Reply:
no, removed